projects/plugins/jetpack/to-test.md

```diff
@@ -1,56 +1,96 @@
-- You should see previews under some of the tweets, and you should not see any notices in your error logs.
+For days, Jack searched for his family, but they were nowhere to be found. Finally, he gave up and resigned himself to the fact that his cooking had somehow turned his family into birds. But despite his grief, he couldn't help but feel a sense of pride. After all, his cooking had been so delicious that it had caused his loved ones to take flight - *quite literally*.
 
-### Pinterest block
+From that day forward, Jack became known as the "bird chef," renowned throughout the land for his incredible burgers and fries that had the power to transform even the most ordinary person into a magnificent bird. And although he never did find his lost family, he took solace in knowing that they were out there somewhere, soaring through the skies, thanks to the power of his cooking.
 
-- Create a new post and add a Pinterest block. When adding a Pinterest URL, no errors should appear in your logs.
+[^1]: Check [the recipe here](https://www.myrecipes.com/recipe/classic-burger).
+
+[^longnote]: This gets weird from here.
+
+    I've been using ChatGPT at times to help me come up with random content, and this is what happens. I think this quote from ["Cheating is All You Need"](https://about.sourcegraph.com/blog/cheating-is-all-you-need) is appropriate here:
+
+    > A raw LLM is like a Harvard CS grad who knows a lot about coding and took a magic mushroom about 4 hours ago, so it’s mostly worn off, but not totally.
+````
+
+### Jetpack Forms
+Jetpack Forms evolve very quickly, and the Form block should get attention from testers in every release. This time there are changes as well. One of them is the update for Forms child blocks to allow any transformation between the blocks:
+
+- Create a post, add a Form block, and select a template
+- Select a child block and use the Transform To option to transform the current block into other blocks
+- You should be able to transform a child block into any other Form child block
+
+Additionally, when themes do not reset the browser styles, the Jetpack button block will display borders on the frontend. To avoid that, we add our own reset, except when the button uses the "outline" styles, where we do want to show a border:
+
+- Start with a site using the Twenty Twenty Two theme.
+- In a new post, add 2 blocks that use the Jetpack button block: the Form block as well as the Revue block are good examples.
+- In one of those blocks, use the default "fill" block style, and in the other use the "outline" style.
+- On the frontend, the button should not have a border when using the "fill" block style.
+
+Another fix was to avoid the following issue. The "Single Choice" and "Multiple Choice" blocks in Jetpack Form blocks, when added to a form inside a Cover block, used to not obey the background setting set in the editor and display with a white background instead. This release fixes the problem:
```

Comment:
```suggestion
A fix has been added to avoid the following issue: the "Single Choice" and "Multiple Choice" blocks in Jetpack Form blocks, when added to a form inside a Cover block, previously did not obey the background setting set in the editor and would display with a white background. This release fixes the problem, to test:
```